diff --git a/node_modules/@vuepress-reco/vuepress-plugin-bgm-player/bin/styles/index.styl b/node_modules/@vuepress-reco/vuepress-plugin-bgm-player/bin/styles/index.styl
index 96912b1..7322a8e 100644
--- a/node_modules/@vuepress-reco/vuepress-plugin-bgm-player/bin/styles/index.styl
+++ b/node_modules/@vuepress-reco/vuepress-plugin-bgm-player/bin/styles/index.styl
@@ -7,7 +7,7 @@
     height 80px
     background-color rgba(255, 255, 255, 0.7)
     background-color var(--default-color-7)
-    border-radius 50px
+    border-radius 10px
     padding 10px
     box-shadow 0 1px 6px 0 rgba(0,0,0,0.2)
     .reco-bgm-cover
